examples/lib/versionManagement.js

```diff
@@ -1,14 +1,28 @@
-// Get the current version number from package.json on GitHub.
+/**
+ * Functions for getting version information.
+ * Note: these functions are not used by the service worker to check for updates;
+ * the service worker updates whenever sw.js has changed.
```

Comment:
How about one more bit here... like:

> ...and sw.js is changed when `grunt ____` is run

? Thank you!

Reply:
@jywarren Sounds good. I have added that change.

Reply:
@jywarren Sounds good. I have added that change.